evm_arithmetization/src/cpu/kernel/tests/account_code.rs

```diff
@@ -139,7 +139,7 @@ pub(crate) fn prepare_interpreter<F: Field>(
     account: &AccountRlp,
 ) -> Result<()> {
     let mpt_insert_state_trie = KERNEL.global_labels["mpt_insert_state_trie"];
-    let check_state_trie = KERNEL.global_labels["check_state_trie"];
+    let check_state_trie = KERNEL.global_labels["check_final_state_trie"];
```

Comment:
I don't think this is even being used? It was probably aimed at being the halt offset but is being overwritten in tests

Reply:
It is used as the initial offset for hashing, is it not? check_txn_trie was used as a halt offset, and I should indeed remove it now, though